### PR TITLE
9446: ESM page copy update

### LINKED ISSUE #9446: Copy updates to ESM page
Hello - The [esm page]( https://ubuntu.com/security/esm) was in dire need of some cosmetic copy updates, so Nikos, Lech and myself have added them into the [copy doc](https://docs.google.com/document/d/1RBZX3PJPvODxRlcnIS10Ii4SkIMH5Fm74EmsMQ0cnic/edit?usp=sharing). 

Nikos accidentally accepted some of the changes and was not able to re-open them, so I have added comments to sections with 'Updated' if these sections' updates were resolved.

Thank you in advance!
Kelly

---
*Reported from: https://ubuntu.com/security/esm*

### PR DESCRIPTION
**Pull request by jpmartinspt**
## Done
Update ESM page copy according to #9446.

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check the copy matches the changes proposed in #9446 and accept the suggestions in the document.

## Issue / Card

Fixes #9446

### CODE REVIEW

File: templates/security/esm.html [hunk 1]
```diff
@@ -96,12 +99,24 @@ <h4>
...
-
+<section class="p-strip is-bordered">
+  <div class="row">
+    <div class="col-8">
+      <h2>Is Ubuntu 16.04 LTS still supported beyond April 2021?</h2>
+      <p>
+        Ubuntu 16.04 LTS will still be supported beyond its free initial five-year maintenance period in April 2021, as it transitions to the extended security maintenance phase - with three additional years of security ensured. 
+      </p>
+      <p>
+        <a href="&nbsp;&rsaquo;">Learn more about Ubuntu 16.04 LTS moving to ESM &nbsp;&rsaquo;</a>
```
Comment (bethcollins92) [inline templates/security/esm.html:110]:
```suggestion
        <a href="/16-04">Learn more about Ubuntu 16.04 LTS moving to ESM &nbsp;&rsaquo;</a>
```

File: templates/security/esm.html [hunk 2]
```diff
@@ -116,8 +131,8 @@ <h2>What's covered?</h2>
       <p>ESM continues security updates for:</p>
       <ul class="u-no-margin--bottom">
         <li>high and critical CVEs (Common Vulnerabilities and Exposures), in</li>
-        <li>the Ubuntu base OS and scale-out infrastructure (Ceph, Openstack, see <a href="https://wiki.ubuntu.com/SecurityTeam/ESM/14.04#A14.04_Infrastructure_ESM_Packages" class="p-link--external">the complete list</a>), on</li>
-        <li>the 64-bit x86 architecture.</li>
+        <li>the Ubuntu base OS and scale-out infrastructure</li>
```
Comment (bethcollins92) [inline templates/security/esm.html:134]: I've added a [comment](https://docs.google.com/document/d/1RBZX3PJPvODxRlcnIS10Ii4SkIMH5Fm74EmsMQ0cnic/edit?disco=AAAAL7XZpsY) to copy doc about these bullet points I feel like they read weird

File: templates/security/esm.html [hunk 3]
```diff
@@ -126,6 +141,19 @@ <h2>What's covered?</h2>
     <hr class="p-separator">
   </div>
 
+  <div class="row">
+    <div class="col-8">
+      <h2>ESM on Azure, AWS and Google Cloud with Ubuntu Pro</h2>
+      <p>
+        If you are running Ubuntu 16.04 LTS images on the public cloud and are looking for continued security coverage with ESM, it is recommended to launch new, Ubuntu Pro images for Azure, AWS and Google Cloud (GCP coming soon). Ubuntu Pro images are paid, premium images that are optimised and priced for the cloud, with security and compliance features built in.
```
Comment (bethcollins92) [inline templates/security/esm.html:148]: The Azure, AWS and Google cloud in the paragraph are also links in the copy doc

File: templates/security/esm.html [hunk 4]
```diff
@@ -134,19 +162,73 @@ <h4 class="u-no-max-width">Do all levels of Ubuntu Advantage for Infrastructure
...
       <h4 class="u-no-max-width">How can we ensure the security of our Ubuntu systems after the end of Standard Security Maintenance?</h4>
-      <p>Sign up to Ubuntu Advantage now, and you will benefit from UA services immediately without having a gap in service when a particular release reaches the end of its Standard Security Maintenance window. For more details of initial release dates and maintenance cadence, see <a href="https://wiki.ubuntu.com/Releases">wiki.ubuntu.com/Releases</a>. Ubuntu Advantage is available through the <a href="/advantage">Ubuntu Advantage portal</a> and through the <a class="p-link--external" href="https://aws.amazon.com/marketplace/pp/B01MZ1LSBQ?qid=1486382973569&sr=0-1&ref_=srh_res_product_title">AWS Marketplace</a>.</p>
+      <p>
+        To ensure security continuity when a particular release reaches the end of its Standard Security Maintenance window, sign up for <a href="/advantage">Ubuntu Advantage</a> for free on up to 3 machines or through a paid subscription for enterprise use. For more details of initial release dates and maintenance cadence, see <a href="https://ubuntu.com/about/release-cycle">https://ubuntu.com/about/release-cycle</a>.
```
Comment (bethcollins92) [inline templates/security/esm.html:166]: Also left another comment about this as we don't normally link to pages like this, I think it should be something like "see Ubuntu's release cycle" or  something similar but we can wait for a response from Lech.

**Review by bethcollins92 — CHANGES_REQUESTED** [review]
Looking good with a few small comments in the code 👍🏼  Might just want to wait to hear back from Lech about that link or Peter is a good person to ask.

File: templates/security/esm.html (hunk 1 above)
Comment (jpmartinspt) [reply templates/security/esm.html]: Fixed.

File: templates/security/esm.html (hunk 3 above)
Comment (jpmartinspt) [reply templates/security/esm.html]: Added the links.

File: templates/security/esm.html (hunk 2 above)
Comment (jpmartinspt) [reply templates/security/esm.html:134]: I removed the `, in` as it does seem like a mistake.

**Pull request merged**